R/Lab 1.Rmd

```diff
@@ -0,0 +1,197 @@
+---
```

Comment:
I think that once you've built the package it should work. Do you remember how to do that?